### PR TITLE
Show message if page was updated since last visit

### LINKED ISSUE #40: See if page has been updated since last viewed
We already show the "last updated date" at the top of the page. I'd like the reader to be able to tell directly if the page has been updated since the page was last viewed.

So basically, we'll need to store the moment of each page it was viewed. Then we can compare these with the "last updated date". If the document has been updated, we can tell the user (highlight the "last updated date" or add a note or tooltip or ...).

### PR DESCRIPTION
**Pull request by vvandijck**
This pull-request includes:
- Shows a message if page was updated since last visit.
- Keeps track of which page(s) and when it was last visited.
- Calculates the days between these 2 dates.

Closes  #40

### CODE REVIEW

**Comment by Siilwyn** [comment]
As discussed, might want to use the `scoped` attribute so no `class` is needed.

File: docs/.vuepress/components/metadata.vue [hunk 1]
```diff
@@ -1,43 +1,117 @@
 <template>
   <header class="metadata" v-if="!isHomePage">
-    <span class="last-updated">
-      Last updated: <time :datatime="prettyDate">{{prettyDate}}</time>
+    <span v-if="updatedSinceLastVisit" class="last-updated last-updated--message">
```
Comment (Siilwyn) [inline docs/.vuepress/components/metadata.vue]: Is it possible to call `hasBeenUpdatedSinceLastVisit` from here directly? The extra `updatedSinceLastVisit` variable seems unnecessary.

The function can be pure, only return the result of `lastVisit < lastUpdate`.

File: docs/.vuepress/components/metadata.vue [hunk 2]
```diff
@@ -1,43 +1,117 @@
...
+      Heads up! This page has been updated since your last visit {{ daysSinceLastVisit }} days ago
```
Comment (Siilwyn) [inline docs/.vuepress/components/metadata.vue]: `daysSinceLastVisit` seems like a computed property?

Comment (Siilwyn) [inline docs/.vuepress/components/metadata.vue]: Can the [Intl.RelativeTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RelativeTimeFormat) be used and fallback to the current code?

Comment (vvandijck) [reply docs/.vuepress/components/metadata.vue]: It is now 👍

Comment (vvandijck) [reply docs/.vuepress/components/metadata.vue]: I don't see the added value of adding this and then also keeping the current code as fallback when this works fine.

**Pull request merged**